2024-03-29 11:41:51.831262
<class 'ValueError'>
ValueError("could not convert string to float: '0.762222},'")
  File "D:\argsdata\TEST\ft_gpt3.5_test.py", line 58, in <module>
    loss(label_score=label_score,score=score,i=i)

  File "D:\argsdata\TEST\ft_gpt3.5_test.py", line 25, in loss
    MAE_loss = abs(float(label_score)-float(score))


